components/create_team/create_team_controller/index.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
```

Comment:
Throughout my PRs, I think I copy and pasted from one file to get this copyright notice started. Recognizing that we're going to do a full sweep of these, I'm optimistic this will all get cleaned up later.